src/components/shortcut/shortcut.tsx

```diff
@@ -0,0 +1,105 @@
+import { Component, Prop, h } from '@stencil/core';
+
+/**
+ * This component can be used on places such as a start page or a dashboard.
+ * Clicking on the component should navigate the user to a new screen,
+ * to which you need to provide a URL, using the `href` property.
+ *
+ * By default, this navigation will happen within the same browser tab.
+ * However, it is possible to override that behavior, using the `target` property.
+ *
+ * @exampleComponent limel-example-shortcut
+ * @exampleComponent limel-example-shortcut-notification
+ * @exampleComponent limel-example-shortcut-styling
+ * @exampleComponent limel-example-shortcut-with-click-handler
+ */
+
+@Component({
+    tag: 'limel-shortcut',
+    shadow: true,
+    styleUrl: 'shortcut.scss',
+})
+export class Shortcut {
+    /**
+     * Name of icon for the shortcut.
+     */
+    @Prop({ reflect: true })
+    public icon: string;
+
+    /**
+     * The text to show below the shortcut. Long label will be truncated.
+     */
+    @Prop({ reflect: true })
+    public label: string;
+
+    /**
+     * The `title` tag of the hyperlink, which can be used to
+     * provide additional information about the link.
+     * It improves accessibility both for sighted users
+     * and users with assistive technologies.
+     */
+    @Prop({ reflect: true })
+    public linkTitle?: string;
+
+    /**
+     * Set to `true` if shortcut is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled: boolean;
+
+    /**
+     * The url that the shortcut leads to.
+     */
+    @Prop({ reflect: true })
+    public href?: string;
+
+    /**
+     * Where to load the linked URL, as the name for a browsing context:
+     * - `_self`: in the current browsing context. (Default)
+     * - `_blank`: in a new tab.
+     * - `_parent`: in the parent browsing context of the current one.
+     * If no parent, behaves as `_self`.
+     * - `_top`: the topmost browsing context (the "highest" context
+     * that's an ancestor of the current one). If no ancestors, behaves as `_self`.
+     */
+    @Prop({ reflect: true })
+    public target: '_self' | '_blank' | '_parent' | '_top' = '_self';
+
+    /**
+     * If specified, will display a notification badge
+     * on the shortcut.
+     */
+    @Prop({ reflect: true })
+    public badge?: number | string;
+
+    public render() {
+        const extendedAriaLabel = this.label + '. ' + this.linkTitle;
```

Comment:
On line 95, we have an `if (this.label)`, indicating that we expect it could be empty. But here, we just assume that it's set, otherwise the label will begin with `. `.

I've added a few tests for how I would expect this to behave. I haven't implemented it, so the tests are currently failing. I was thinking perhaps you could do the implementation @LucyChyzhova? 😊

Reply:
Thank Adrian! Sure I will add this 🤗